### DISCUSSION TITLE
Production Rdy?

### DISCUSSION THREAD

**Discussion by yildizbilal** [yonahd/kor#79] [Q&A] [answered]
Hey guys, 

i appreciate your work. I need such a tool to clean three clusters. Did you test the code and do you can recommend this tool to cleanup production clusters?

Best regards

**Comment by yonahd** [comment, marked as answer]
Hi @yildizbilal
A couple important points to look at before production.

1. The list of false positives https://github.com/yonahd/kor#Supported-resources-and-limitations and the general idea of these.
2. The tool currently does not offer a delete flag.

**Comment by yildizbilal** [comment]
Hi @yonahd,

thanks for the hints.

Im just wondering, if it can be possible to implement a solution that finds also references in custom resources by using not exact search. More in a pattern way with regex's (or something like that). 

Following feature can also be interesting: To show in which resource the reference is set could be shown (optional by passing a flag). So then if a resource is found e.g. in a custom resource, then the user can faster check the file if the found reference is correct.

**Reply by yonahd** [reply]
@yildizbilal not sure I understand the first suggestion.

Regarding the second idea, that's a great one, the problem is this tool is doing the reverse, finding the unreferenced ones. Did I understand your idea correctly?
